lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -1936,7 +1936,7 @@ DefaultTextEditingStrategy createDefaultTextEditingStrategy(HybridTextEditing te
     strategy = IOSTextEditingStrategy(textEditing);
   } else if (browserEngine == BrowserEngine.webkit) {
     strategy = SafariDesktopTextEditingStrategy(textEditing);
-  } else if (browserEngine == BrowserEngine.blink &&
+  } else if ((browserEngine == BrowserEngine.blink || browserEngine == BrowserEngine.firefox) &&
```

Comment:
Is there a contextual reason why we have to specify a browser engine as well as the OS when we're trying to select for a mobile text editing strategy? Or can we just always choose the android strategy when `OS == android` and always choose ios strategy when `OS == iOS`